settings.html

```diff
@@ -13,7 +13,7 @@
 </head>
 
 <body>
-	<a href="index.html" id="back" aria-label="Home Page">
+	<a href="/" id="back" aria-label="Home Page">
```

Comment:
Whats the reasoning behind this change? As far as I am aware these do the same things, except linking to `/` will require that you run a web server when testing locally which can be annoying. `index.html` should just work out of the box though.

Reply:
I just prefer it not saying `index.html` at the end of the url, and I also noticed that

Reply:
In that case, this commit probably should not be included